wheretofindme/migrations/0005_follow.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.5 on 2019-01-11 17:52
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wheretofindme', '0004_auto_20190109_0719'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Follow',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
Does this need a distinct ID column? The pair (`from_user`, `to_user`) is a good natural key (and should be made unique, regardless).

Reply:
Django implicitly makes an `id` unless you label something as a PK, and I have learned that it helps you avoid more pain than not. I should, however, add a `unique_together` constraint.